system-addon/content-src/components/TopSites/TopSiteForm.jsx

```diff
@@ -117,7 +119,7 @@ export class TopSiteForm extends React.PureComponent {
         <section className="edit-topsites-inner-wrapper">
           <div className="form-wrapper">
             <h3 className="section-title">
-              <FormattedMessage id={this.props.editMode ? "topsites_form_edit_header" : "topsites_form_add_header"} />
+              <FormattedMessage id={this.props.url ? "topsites_form_edit_header" : "topsites_form_add_header"} />
```

Comment:
Let's pull out `this.props.url` to be clear that this decision matches up with the button text below.
```js
// For UI purposes, editing without an existing link is "add"
const showAsAdd = !this.props.url;
```